src/main/java/com/myaudiolibrary/web/model/Artist.java

```diff
@@ -0,0 +1,58 @@
+package com.myaudiolibrary.web.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import javax.persistence.*;
+import java.util.Set;
+
+// Annoter la classe Artist en tant qu'entité pour qu'elle puisse récupérer les données de la table artist
+@Entity
+public class Artist {
+
+    // Annoter champ id de manière a ce qu'il puisse gérer les identifiants générés automatiquement par MySQL.
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "ArtistId")
+    private Integer id;
+
+    @Column(name = "Name")
+    private String name;
+
+    @JsonIgnoreProperties("artist")
+    @OneToMany(mappedBy = "artist", fetch = FetchType.EAGER, cascade = CascadeType.REMOVE)
+    private Set<Album> albums;
```

Comment:
```suggestion
    private Set<Album> albums = new HashSet<>();
```
HashCode important pour assurer l'unicité !